### ISSUE TITLE
error to call opencv

### ISSUE THREAD

**Issue by longqian95** [stevengj/PyCall.jl#248]
The following simple code has error: 

> ERROR: PyError (:PyObject_Call) <type 'exceptions.TypeError'>
> TypeError('Layout of the output array image is incompatible with cv::Mat (step[ndims-1] != elemsize or step[1] != elemsize*nchannels)',)

```julia
using PyCall
@pyimport cv2
cv2.findContours(rand(UInt8,64,64),cv2.RETR_EXTERNAL,cv2.CHAIN_APPROX_SIMPLE)
```

But the corresponding python code has no problem:

```python
import cv2
cv2.findContours(array(randint(0,255,(64,64)),uint8),cv2.RETR_EXTERNAL,cv2.CHAIN_APPROX_SIMPLE)
```

Julia: 0.4.3
PyCall: master
python: 2.7
opencv: 2.4.x
os: ubuntu 14.04

**Comment by stevengj** [comment]
It sounds like OpenCV is expecting an array row-major order, whereas Julia's array is in column-major order.  This needs something like #85, it seems like.

**Comment by stevengj** [comment]
Fixed by #265

**Comment by longqian95** [comment]
Thanks. Now cv2.findContours(PyReverseDims(rand(UInt8,64,64)), cv2.RETR_EXTERNAL, cv2.CHAIN_APPROX_SIMPLE) works.
